### PR TITLE
changes for license protection report v2

### PR DESCRIPTION
**Pull request by anandab-msft**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/information-protection-license-report.md [hunk 1]
```diff
@@ -32,17 +33,37 @@ You must be a Teams service admin to make these changes. See [Use Teams administ
...
+|**10**   |**Change Notification API Usage**<li>**Display Name** - Display name of the user against which a change notification was triggered.</li><li>**Required License** - Whether the given user has the required license to successfully send a change notification to the app.</li><li>**Attempted Count** - Total number of change notifications that were triggered because of this user.</li><li>**Change Notification Sent** - Total number of change notifications that were sent to the app. This will be less than the total change notifications triggered if the user doesn't have a valid license.</li>|
+|**11**|**Patch API**<li>**Display Name** - Display name of the user where an attempt to update the message was done.</li> <li>**Required License** - Whether the given user has the required license for the message to get succssfully updated.</li><li>**Attempted Count** - Total attempts to update messages.</li><li>**Message Patched** - Total message that was successfully updated.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/information-protection-license-report.md:63]:
```suggestion
|**11**|**Patch API**<li>**Display Name** - Display name of the user where an attempt to update the message was done.</li> <li>**Required License** - Whether the given user has the required license for the message to get succssfully updated.</li><li>**Attempted Count** - Total attempts to update messages.</li><li>**Message Patched** - Total messages that were successfully updated.</li>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @anandab-msft, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by anandab-msft** [comment]
thanks @JohanFreelancer9 @scanum  - can you mark this as with "do not merge " label (if there is such label). I intend to get this published in 2nd week of Dec.

**Comment by anandab-msft** [comment]
@scanum - can you pls merge this PR

**Comment by kapilchhajer** [comment]
@scanum / @JohanFreelancer9  please can you review changes.

**Comment by MicrosoftHeidi** [comment]
I've reviewed and am approving now.

**Pull request merged**